src/Client/components/ques/mcq-ques.tsx

```
@@ -56,14 +93,56 @@ export default function App() {
-        correctOption: 1,
+        correctOptions: [],
+        points: 0,
+        difficultyType: '',
       },
     ])
     setCurrentPage(questions.length)
   }
 
+  const handlePoints = (questionIndex: number, value: string) => {
+    const newQuestions = [...questions]
+    newQuestions[questionIndex].points = parseInt(value) || 0
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)